README.md

```diff
@@ -157,6 +157,91 @@ xtf.foo.v2.version=1.0.3
 
 Retrieving an instance with this metadata: `Produts.resolve("product");`
 
+#### Service Logs Streaming (SLS)
+This feature allows for you to stream the services output while the test is running; this way you can see immediately 
+what is happening inside the cluster.
+This is of great help when debugging provisioning, specifically on Cloud environments, which instead would require for
+you to access your Pods.
+
+##### Kubernetes/OpenShift implementation
+The SLS OpenShift platform implementation relies upon the following fabric8 Kubernetes Client API features:
+
+- Watching Kubernetes events (see
+  [PodWatchEquivalent.java](https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/kubectl/equivalents/PodWatchEquivalent.java))
+- Watching Pod logs (see
+  [PodLogExample.java](https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/PodLogExample.java))
+
+The expected behavior is to stream the output of all the containers that are started or terminated in the selected 
+namespaces.
+
+##### Usage
+The SLS feature can be configured and enabled either via annotations or via properties.
+This behavior is provided by 
+[the ServiceLogsStreamingRunner JUnit 5 extension](./junit5/src/main/java/cz/xtf/junit5/extensions/ServiceLogsStreamingRunner.java).
+There are two different ways for enabling the SLS functionality, which are summarized in the following sections, please 
+refer to the [JUnit 5 submodule documentation](./junit5/README.md) in order to read about the extension implementation 
+details.
+
+###### The `@ServiceLogsStreaming` annotation (Developer perspective)
+Usage is as simple as annotating your test with `@ServiceLogsStreaming` e.g.:
+
+```java
+@ServiceLogsStreaming
+@Slf4j
+public class HelloWorldTest {
+  // ...
+}
+```
+
+###### The `xtf.sls.enabled` and `xtf.sls.config` property (Developer/Automation perspective)
+You can enable the SLS feature by setting the `xtf.sls.enabled` property so that it would apply to 
```

Comment:
Just a suggestion. `sls` seems to hard to remember (at least to me :-) ) Could it be renamed to `xtf.log.streaming.enabled` ? wdyt?

Reply:
Thanks @mnovak1, will do.